src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,35 +37,37 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${selectedTodo.id}`}
```

Comment:
```suggestion
              {`Todo #${id}`}
```

Reply:
Sorry, I didn't get this one...
How could it be just `id`?...